demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -581,6 +586,46 @@ def validate_modified_files(self, modified_files):
                                                          is_modified=True, old_file_path=old_file_path))
         return all(valid_files)
 
+    def run_dependencies_validations_on_pack(self, pack, id_set_path=None):
+        core_pack_list = tools.get_remote_file('Tests/Marketplace/core_packs_list.json') or []
+        first_level_dependencies = PackDependencies.find_dependencies(
+            pack, id_set_path=id_set_path, silent_mode=False, exclude_ignored_dependencies=False,
+            update_pack_metadata=False)
+
+        for core_pack in core_pack_list:
+            first_level_dependencies.pop(core_pack, None)
+        if not first_level_dependencies:
+            return True
+
+        dependency_result = json.dumps(first_level_dependencies, indent=4)
+        click.echo(click.style(f"Found dependencies result for {pack} pack:", bold=True))
+        click.echo(click.style(dependency_result, bold=True))
+
+        if first_level_dependencies.get('NonSupported') or first_level_dependencies.get('DeprecatedContent'):
+            error_message, error_code = Errors.invalid_package_dependencies(pack)
+            if self.handle_error(error_message, error_code, file_path=pack):
+                return False
+            return False
```

Comment:
if `handle_error` return false, it means the error should be ignored.
```suggestion
            if self.handle_error(error_message, error_code, file_path=pack):
                return False
```